actions/views/root.js

```diff
@@ -1,32 +1,12 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import {getClientConfig, getLicenseConfig} from 'mattermost-redux/actions/general';
-import * as UserActions from 'mattermost-redux/actions/users';
 import {Client4} from 'mattermost-redux/client';
 
 import {ActionTypes} from 'utils/constants';
 import en from 'i18n/en.json';
 import {getCurrentLocale, getTranslations} from 'selectors/i18n';
 
-export function loadMeAndConfig() {
```

Comment:
Why get rid of `loadMeAndConfig`? Generally, I'd like to move more logic like this out of components and into actions since it helps keep the components a bit simpler since they tend to have a lot of logic in them. It also helps us reduce the number of props being passed into the component since that has a slight affect on performance

Reply:
i replaced the function back and moved the actions to that thunk function itself